pennylane/operation.py

```diff
@@ -636,16 +646,30 @@ def get_parameter_shift(self, idx):
         """
         # get the gradient recipe for this parameter
         recipe = self.grad_recipe[idx]
-        multiplier, shift = (0.5, np.pi / 2) if recipe is None else recipe
+
+        # Default values
+        multiplier = 0.5
+        a = 1
+        shift = np.pi / 2
+
+        # We set the default recipe following:
+        # ∂f(x) = c1*f(a1*x+s1) + c2*f(a2*x+s2)
+        # where we express a positive and a negative shift by default
+        default_param_shift = [[multiplier, a, shift], [-multiplier, a, -shift]]
+        param_shift = default_param_shift if recipe is None else recipe
 
         # internal multiplier in the Variable
         var_mult = self.data[idx].mult
 
-        multiplier *= var_mult
-        if var_mult != 0:
-            # zero multiplier means the shift is unimportant
-            shift /= var_mult
-        return multiplier, shift
+        for elem in param_shift:
+
+            # Update the multiplier
+            elem[0] *= var_mult
+            if var_mult != 0:
+                # Update the shift
+                # zero multiplier means the shift is unimportant
+                elem[2] /= var_mult
+        return param_shift
```

Comment:
Apart from this block here, it's a shame we can't re-use this method for the tape-mode gradient 🙁 

Do you think it makes sense to do the following?

```python
def get_parameter_shift(self, idx, shift=np.pi/2):
    recipe = self.grad_recipe[idx]

    # Default values
    multiplier = 0.5 / np.sin(s)
    a = 1

    # We set the default recipe following:
    # ∂f(x) = c1*f(a1*x+s1) + c2*f(a2*x+s2)
    # where we express a positive and a negative shift by default
    default_param_shift = [[multiplier, a, shift], [-multiplier, a, -shift]]
    param_shift = default_param_shift if recipe is None else recipe

    if hasattr(self.data[idx], "mult"):
        # Parameter is a variable, we are in non-tape mode
        ...

    return param_shift
```

This way:

* We can call this method inside both the old QNode gradient methods, and the new tape gradient methods.

* We can pass a different default shift value, for the qubit parameter-shift rule

* The branch that worries about variables only is called in non-tape mode. Note that the above is safer than using `qml.tape_active()`, which only checks if the user has activated tape mode, not if the object itself is a tape.

Reply:
Hi Josh, thanks so much (forgot to reply here :sweat_smile:). Worked great, adjusted the method!